launch_testing_ros/launch_testing_ros/wait_for_topics.py

```diff
@@ -131,6 +155,13 @@ def __init__(
         self.expected_topics = set()
         self.received_topics = set()
         self.received_messages_buffer = {}
+        self._any_publisher_connected = Event()
```

Comment:
We can probably drop the `_` for this, it's basically just like `msg_event_object`, i.e., public for `_WaitForTopicsNode`, but otherwise private:

```suggestion
        self.any_publisher_connected = Event()
```